docs/java-compatibility.adoc

```diff
@@ -0,0 +1,8 @@
+= Java Compatibility with TomEE 8.0.0-M3
+:index-group: General Information
+:jbake-type: page
+:jbake-status: published
+
+TomEE 8.0.0-M3 is Java EE 8 & Java 11. Any OS that supports Java 11 works
```

Comment:
As far as I know, TomEE 8.0.0-M3 is still not built using Java 11.

Reply:
So, TomEE 8.0.0-M3 is only Java EE 8 as of now?

Reply:
But it works fine @ runtime with Java 11even if it is not build with it :)

Reply:
Hi @rzo1 , Base on a couple of thread on the mailing list [1]. Current TomEE M3 and therefore master branch doesn't fully compile nor pass yet all the test when runs on Java 11. The impression TomEE M3 runs on Java 11 may be true for a couple of apps but the server is not yet fully Java 11 compatible. That's why I think advertising on the website that TomEE M3 is Java 11 compatible may be misleading for the reader.

[1] https://lists.apache.org/list.html?dev@tomee.apache.org:lte=6M:java%2011

Reply:
@cesarhernandezgt please check the document where I made changes, if the statements under Note are good. Or suggest a better notes which can be published

Thanks

Reply:
done, I marked as +1, thanks much :)